requirements/supported-sharepoint-editions.md

```diff
@@ -1,10 +1,9 @@
 ---
 title: Supported SharePoint Editions
 ---
-# Supported SharePoint Editions
-
 The following SharePoint editions and additional products are supported by SPDocKit:
 
+* SharePoint Online
 * SharePoint Server 2016
 * SharePoint 2013
   * Microsoft SharePoint Foundation 2013
```

Comment:
SharePoint 2007 is no longer __a__ supported edition, starting with SPDockit 6. If you wish to document and explore your SharePoint 2007 farms, please use SPDocKit 5.